feature/static/otg_tests/next_hop_encap_udp/static_encap_gue_ipv4_test.go

```diff
@@ -0,0 +1,1176 @@
+	trafficDstNetIPv6  = "2001:DB8::10:1:1:1"
+
+	// Random Source Address
+	trafficSrcNetIPv4 = "1.1.1.1"
+	trafficSrcNetIPv6 = "2001:db8::aaaa:0"
+
+	// Lag config
+	lag1Name    = "lag1"
+	lag2Name    = "lag2"
+	dutLag1Name = "Port-Channel1"
+	dutLag2Name = "Port-Channel2"
+
+	frameSize          = 512
+	packetCount        = 12000000
+	gueProtocolPort    = 6080
+	gueSrcProtocolPort = 5000
+
+	tolerancePct     = 6.0
+	tolerance        = 0.06
+	bgpPeerGroupName = "BGP-PEER-GROUP"
+)
+
+var (
+	atePort1 = &attrs.Attributes{
+		Name:    "atePort1",
+		IPv4:    "192.168.10.1",
+		IPv4Len: plenIPv4,
+		IPv6:    "2001:db8::192:168:10:1",
+		IPv6Len: plenIPv6,
+		MAC:     "02:00:00:00:00:01",
+	}
+
+	dutPort1 = attrs.Attributes{
+		Desc:    "DUT to ATE Port1",
+		IPv4:    "192.168.10.2",
+		IPv4Len: plenIPv4,
+		IPv6:    "2001:db8::192:168:10:2",
+		IPv6Len: plenIPv6,
+		MAC:     "00:1a:11:00:00:02",
+	}
+
+	ateLAG1 = &attrs.Attributes{
+		Name:    "ateLag1",
+		IPv4:    "192.168.20.1",
+		IPv4Len: plenIPv4,
+		MAC:     "02:00:00:00:01:01",
+	}
+
+	dutLAG1 = attrs.Attributes{
+		Desc:    "DUT to ATE LAG1",
+		IPv4:    "192.168.20.2",
+		IPv4Len: plenIPv4,
+	}
+
+	ateLAG2 = &attrs.Attributes{
+		Name:    "ateLag2",
+		IPv4:    "192.168.30.1",
+		IPv4Len: plenIPv4,
+		MAC:     "02:00:00:00:02:01",
+	}
+
+	dutLAG2 = attrs.Attributes{
+		Desc:    "DUT to ATE LAG2",
+		IPv4:    "192.168.30.2",
+		IPv4Len: plenIPv4,
+	}
+
+	otgPorts = map[string]*attrs.Attributes{
+		"port1": atePort1,
+	}
+
+	dutPorts = map[string]*attrs.Attributes{
+		"port1": &dutPort1,
+	}
+)
+
+var capturePorts = []string{"port2", "port3", "port4", "port5"}
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
+	t.Helper()
+	d := gnmi.OC()
+
+	p1 := dut.Port(t, "port1")
+	gnmi.Replace(t, dut, d.Interface(p1.Name()).Config(), dutPort1.NewOCInterface(p1.Name(), dut))
+
+	// Ports 2 and 3 will be part of LAG
+	dutAggPorts1 := []*ondatra.Port{
+		dut.Port(t, "port2"),
+		dut.Port(t, "port3"),
+	}
+	configureDUTLag(t, dut, dutAggPorts1, dutLag1Name, dutLAG1)
+
+	// Ports 4 and 5 will be part of LAG
+	dutAggPorts2 := []*ondatra.Port{
+		dut.Port(t, "port4"),
+		dut.Port(t, "port5"),
+	}
+	configureDUTLag(t, dut, dutAggPorts2, dutLag2Name, dutLAG2)
+
+	root := &oc.Root{}
+	ni := root.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut))
+
+	t.Log("Configuring BGP")
+	bgpProtocol := ni.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP")
+	bgp := bgpProtocol.GetOrCreateBgp()
+	g := bgp.GetOrCreateGlobal()
+	g.As = ygot.Uint32(dutAS)
+
+	pg := bgp.GetOrCreatePeerGroup(bgpPeerGroupName)
+	pg.PeerAs = ygot.Uint32(ateAS)
+
+	ipv6Nbr := bgp.GetOrCreateNeighbor(atePort1.IPv6)
+	ipv6Nbr.PeerGroup = ygot.String(bgpPeerGroupName)
+	ipv6Nbr.PeerAs = ygot.Uint32(ateAS)
+	ipv6Nbr.Enabled = ygot.Bool(true)
+	ipv6Nbr.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST).Enabled = ygot.Bool(true)
```

Comment:
Please use the [cfgplugin for bgp](https://github.com/openconfig/featureprofiles/blob/main/internal/cfgplugins/bgp.go)

Reply:
In cfgplugin.bgp.go, the func NewBGPSession uses a minimum of 2 ports. According to the current test plan, we are having one port as an ingress and egress port as LAG ports, skipping the use of the already existing function